### PR TITLE
feat(application): UI for collect signatures

### PR DESCRIPTION
**Pull request by siggakogk**
# \<Description\>

UI for collect signatures step in party letter application 

## Screenshots / Gifs

image

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by stefanhar — APPROVED** [review]
LGTM

**Review by barabrian — APPROVED** [review]
LTGM

File: libs/application/templates/party-letter/src/lib/messages.ts [hunk 1]
```diff
@@ -104,7 +104,7 @@ export const m: MessageDir = {
     partyLetterSubtitle: {
       id: 'ple.application:partyletter.subtitle',
       defaultMessage:
-        'Við  alþingiskosningarnar 28. október 2017 buðu eftirtalin stjórnmálasamtök fram lista og voru þeir merktir sem hér segir:',
+        'Við  alþingiskosningarnar 28. október 2017 buðu eftirtalin stjórnmálasamtök fram lista og voru þeir merktir sem hér segir:',
```
Comment (jeremybarbet) [inline libs/application/templates/party-letter/src/lib/messages.ts:107]: extra whitespace here?

File: libs/application/templates/party-letter/src/lib/messages.ts [hunk 2]
```diff
@@ -204,4 +204,61 @@ export const m: MessageDir = {
...
+    },
+    nameInput: {
+      id: 'ple.application:collect.name.input',
+      defaultMessage: 'Nafn',
+      description: 'Label and placeholder for name input field',
+    },
+    descriptionPt1: {
+      id: 'ple.application:collect.descripton.pt.one',
+      defaultMessage:
+        'Með því að mæla með úthlutun tiltekins listabókstafs til tilgreinds stjórnamálaflokks  samþykkir þú að viðkomandi stjórnmálaflokkur, dómsmálaráðuneytið og Þjóðskrá Íslands fái aðgang að þeim upplýsingum sem skráðar eru. Þeir aðilar hafa ekki heimild til að miðla þeim upplýsingum frekar.',
```
Comment (jeremybarbet) [inline libs/application/templates/party-letter/src/lib/messages.ts:232]: extra whitespace here `stjórnamálaflokks   samþykkir`

File: libs/application/templates/party-letter/src/lib/messages.ts (hunk 1 above)
Comment (siggakogk) [reply libs/application/templates/party-letter/src/lib/messages.ts:107]: ahh just text copied from figma :) i will fix it

**Review by jeremybarbet — APPROVED** [review]

**Pull request merged**